apps/condo/pages/user/index.tsx

```diff
@@ -35,31 +38,58 @@ export const UserInfoPageContent: React.FC<IUserInfoPageContentProps> = ({ organ
     const UpdateMessage = intl.formatMessage({ id: 'Edit' })
     const InterfaceLanguageTitle = intl.formatMessage({ id: 'pages.condo.profile.interfaceLanguage' })
     const ChooseInterfaceLanguageTitle = intl.formatMessage({ id: 'pages.condo.profile.chooseInterfaceLanguage' })
+    const GlobalHintsTitle = intl.formatMessage({ id: 'pages.condo.profile.globalHints' })
     const RuTitle = intl.formatMessage({ id: 'language.russian.withFlag' })
     const EnTitle = intl.formatMessage({ id: 'language.english-us.withFlag' })
 
+    const locale = intl.locale
+
+    const [showGlobalHints, setShowGlobalHints] = useState<boolean>(false)
+
     const { user, refetch } = useAuth()
     const userOrganization = useOrganization()
     const { breakpoints } = useLayoutContext()
 
-    useEffect(() => {
-        refetch()
-    }, [refetch])
+    const updateUser = User.useUpdate({})
 
     const name = get(user, 'name')
     const email = get(user, 'email', '')
+    const phone = get(user, 'phone')
+    const initialShowGlobalHints = get(user, 'showGlobalHints')
 
     const possibleLocalesOptions = useMemo(() => ([
         { label: RuTitle, value: 'ru' },
         { label: EnTitle, value: 'en' },
     ]), [EnTitle, RuTitle])
 
-    const updateUser = User.useUpdate({})
-
-    const localeChangeHandler = useCallback((setLocale) => async (newLocale) => {
+    const handleLocaleChange = useCallback((setLocale) => async (newLocale) => {
         await updateUser({ locale: newLocale }, { id: user.id })
         setLocale(newLocale)
-    }, [updateUser, user.id])
+    }, [user.id])
+
+    const updateGlobalHints = useMemo(() => debounce(async (checked) => {
+        try {
+            await updateUser({ showGlobalHints: checked }, { id: user.id })
+            refetch()
+        } catch (e) {
+            console.error('Failed to update field "showGlobalHints"')
+        }
+    }, 400), [user.id])
+
+    const handleGlobalHintsChange: SwitchChangeEventHandler = useCallback(async (checked) => {
+        setShowGlobalHints(checked)
+        await updateGlobalHints(checked)
+    }, [updateGlobalHints])
+
+    useEffect(() => {
```

Comment:
why we need a front-end refetch?

Reply:
When we go to user page, we call `refetch` to get actual data - this is old logic, it has already been